python/ml4ir/config/parse_args.py

```diff
@@ -203,10 +203,17 @@ def define_args() -> ArgumentParser:
     parser.add_argument(
         "--group_metrics_min_queries",
         type=int,
-        default=50,
+        default=25,
         help="Minimum number of queries per group to be used to computed groupwise metrics.",
     )
 
+    parser.add_argument(
+        "--gradient_clip_value",
+        type=float,
+        default=5.0,
```

Comment:
this 5.0 should just reference a constant in the optimizers module, perhaps?

Reply:
Hmm. That way we have the optimizer module kinda self-contained and easily readable?